guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

```diff
@@ -60,8 +60,8 @@ Check the size of /var/lib/pulp/published with 'du -sh /var/lib/pulp/published/'
 # {foreman-maintain} content prepare
 ----
 +
-. As part of the final step of `{foreman-maintain} content prepare`, {Project} checks whether any content units are unmigrated.
-If `{foreman-maintain} content migration-stats` identifies corrupted or missing content, you might see something similar to the following:
+. By the end of its run, the command checks whether any content units are unmigrated.
```

Comment:
Please, remove the `. ` at the start of the line, because it's not a separate step, but it's another paragraph that is part of the preceding step.